tests/factories.py

```diff
@@ -72,6 +72,7 @@ class Meta:
     description = factory.Faker("word")
     cpe = "cpe:/o:redhat:enterprise_linux:9"
     tag = factory.RelatedFactory(ProductStreamTagFactory, factory_related_name="product_stream")
+    active = True
```

Comment:
Should we default this to a random value, so we can accurately test latest vs. non-latest streams without hardcoding the behavior? Or maybe it should be latest=False to test the non-latest behavior by default, and force facotry users to opt-in if they want to work with a latest stream.